app/gui2/src/stores/graph/graphDatabase.ts

```diff
@@ -111,6 +112,8 @@ export class BindingsDb {
 
 export class GraphDb {
   nodeIdToNode = new ReactiveDb<ExprId, Node>()
+  nodeRects = reactive(new Map<ExprId, Rect>())
+  vizRects = reactive(new Map<ExprId, Rect>())
```

Comment:
So far, GraphDB was focusing on node's "Logic", not how they are displayed, and I liked it. But if we want to merge those concepts, perhaps the rect and visRect should be fields of `Node`?

Reply:
I also don't like this change. The bounding boxes should stay at the store level. Also the clearing mechanism should be performed by node views, not as part of the `delete node` operation.